server/util/redact/redact_test.go

```diff
@@ -149,15 +149,15 @@ func TestRedactMetadata_OptionsParsed_StripsURLSecretsAndRemoteHeaders(t *testin
 		CmdLine: []string{
 			"213wZJyTUyhXkj381312@foo",
 			"--flag=@repo//package",
-			"--remote_header=x-buildbuddy-platform.container-registry-password=TOPSECRET",
+			"--remote_exec_header=x-buildbuddy-platform.container-registry-password=TOPSECRET",
 			"--bes_header=foo=TOPSECRET",
 			"--build_metadata=PATTERN=@//foo,NAME=@bar,SECRET=TOPSECRET@",
 			"--some_other_flag=SUBFLAG=@//foo",
 		},
 		ExplicitCmdLine: []string{
 			"213wZJyTUyhXkj381312@explicit",
 			"--flag=@repo//package",
-			"--remote_header=x-buildbuddy-platform.container-registry-password=TOPSECRET_EXPLICIT",
+			"--remote_exec_header=x-buildbuddy-platform.container-registry-password=TOPSECRET_EXPLICIT",
```

Comment:
For this test I think we want to test for both `--remote_header` and `--remote_exec_header` right?

Reply:
good catch - done